cmd/adm.go

```diff
@@ -71,9 +61,15 @@ var updateRefCmd = &cobra.Command{
 	RunE:  SetRefName,
 }
 
-var defaultApiClusterCmd = &cobra.Command{
+var defaultApiClusterCmd = &cobra.Command{ //deprecated is being replaced with "apiClusterCmd"
 	Use:   "default-apicluster <cluster>",
-	Short: `Set configured default API cluster for ao.`,
+	Short: `Set configured API cluster for ao.`,
+	RunE:  SetApiCluster,
+}
+
+var apiClusterCmd = &cobra.Command{
+	Use:   "apicluster <cluster>",
+	Short: `Set configured API cluster for ao.`,
```

Comment:
```suggestion
	Short: `Set configured API cluster for ao.`,
	Aliases: []string{"default-apicluster"}, // deprecated
```
Kan `default-apicluster` legges inn som alias på den nye kommandoen og fjerne `defaultApiClusterCmd`?

Reply:
Der lærte jeg noe nytt.  Jeg har lett i dok for spf13/cobra etter alias for kommandoer uten å finne noe tidligere. Det skal absolutt forsøkes.